wireless/wapi/src/util.c

```diff
@@ -148,7 +148,7 @@ static bool wapi_json_update(FAR cJSON *root,
           int len = strlen(obj->valuestring);
           if (len > 0)
             {
-              if (!strncmp(value, obj->valuestring, len))
+              if (!strcmp(value, obj->valuestring))
```

Comment:
could you please add spaces at line 159?

Reply:
> could you please add spaces at line 159?

Can you give me an example, Line 159 seems pretty normal to me...am I missing something？

Reply:
Oops. It seems that it was a broken preview from my browser. It looked like 159 line was missing 2 spaces before `{`. I looked there again and it seems like all is fine.